docs/api.md

```diff
@@ -0,0 +1,112 @@
+# API documentation
+
+## Event model
+
+Event is a main class used by `django-audittrail` to store events in database for your application.
+This model contains those fields:
+
+`user_id`
+
+* type: PositiveIntegerField
+* required: False
+* description: ID from `User` model.
+
+
+`user_description`
+
+* type: CharField
+* required: True
+* default: `''`
+* description: `User` description.
+
+
+`ip_addr`
+
+* type: GenericIPAddressField
+* required: True
+* description: The IP address of the client.
+
+
+`event_time`
+
+* type: DateTimeField
+* required: True
+* auto_now: True
+* description: Date when event was created.
+
+
+`request_path`
+
+* type: URLField
+* required: True
+* description: URL of called request.
+
+
+`event_description`
+
+* type: TextField
+* required: True
+* description: Description of action done by user.
+
+
+`content_object`
+
+* type: GenericForeignKey
+* required: False
+* description: `GenericForeignKey` was used to attach audit trail events to any object in the system.
+
+By default `Event` model is not registered in Django admin site to changed that you have to set `ENABLE_EVENT_ADMIN` on `True` in your setting file.
```

Comment:
"registered in Django admin site. To enable edition set `ENABLE_EVENT_ADMIN` to `True` in your Django settings module".